src/Connector.js

```diff
@@ -162,7 +162,7 @@ function parseULMessage(topic, message, devices) {
 
   return {
     id,
-    entityId,
+    sensorId: parseInt(message.entityId, 10),
```

Comment:
I think that message is a String since you've done a `split` on it.
`parseInt(entityId, 10),`